ui/src/timeMachine/components/TagSelector.tsx

```diff
@@ -49,6 +48,9 @@ import {
 
 const SEARCH_DEBOUNCE_MS = 500
 
+// We don't show these columns in results but they're able to be grouped on
```

Comment:
This is not always true.  There are scenarios where a `_start` and `_stop` are not not columns in a query response.

Reply:
Russ mentioned that, but it seems like in the context of query builder, it's safe to assume we can group on those columns? I'll change the comment